src/targets/anyleaf_2400.ini

```diff
@@ -0,0 +1,33 @@
+# ********************************
+# AnyLeaf Receiver
+# ********************************
+
+[env:Anyleaf_RX_single_radio_via_UART]
+extends = env:Unified_ESP8285_2400_RX_via_UART
+board_config = anyleaf.rx_2400.single-radio
+
+[env:Anyleaf_RX_single_radio_via_BetaflightPassthrough]
+extends = env:Anyleaf_RX_single_radio_via_UART
+
+[env:ANYLEAF_RX_single_radio_via_WIFI]
+extends = env:Anyleaf_RX_single_radio_via_UART
+
+[env:Anyleaf_RX_dual_radio_via_UART]
+extends = env:Unified_ESP32_2400_RX_via_UART
+board_config = anyleaf.rx_2400.dual-radio
+
+[env:Anyleaf_RX_dual_radio_via_BetaflightPassthrough]
+extends = env:Anyleaf_RX_dual_radio_via_UART
+
+[env:Anyleaf_RX_dual_radio_via_WIFI]
+extends = env:Anyleaf_RX_dual_radio_via_UART
+
+[env:Anyleaf_H7_FC_RX_via_UART]
+extends = env:Unified_ESP8285_2400_RX_via_UART
+board_config = anyleaf.rx_2400.h7-fc
+
+[env:Anyleaf_H7_FC_RX_via_BetaflightPassthrough]
+extends = env:ANYLEAF_H7_FC_RX_via_UART
+
+[env:Anyleaf_H7_FC_RX_via_WIFI]
+extends = env:ANYLEAF_H7_FC_RX_via_UART
```

Comment:
We don't really need these as we can choose the generic versions and provide the device name in the configurator as an alias.

Reply:
Should the file be deleted? The contents changed? Thanks.

Reply:
It can be totally removed.
When flashing from platformio, choose target "Unified ..." and at the end of the build a menu is presented and you can choose the layout to append to the binary.